### DISCUSSION TITLE
Can one set a password to be used with Redis?

### DISCUSSION THREAD

**Discussion by lrhazi-cua** [voiio/dramatiq-crontab#48] [Q&A] [answered]

**Comment by lrhazi-cua** [comment, marked as answer]
I think I found  a way:
```

DRAMATIQ_CRONTAB = {
    "REDIS_URL": f"redis://:{env('REDIS_HOST_PASSWORD')}@redis:6379/0",
}
```

**Comment by codingjoe** [comment]
Hi @lrhazi-cua,

the service URL notation does in deed allow for credentials to be passed. Your solution looks right.

You can either compose the URL like you did in your example, or simply construct the URL and store it in a safe environment.

Cheers
Joe
